go/chat/server.go

```diff
@@ -1124,6 +1124,9 @@ func (h *Server) DownloadAttachmentLocal(ctx context.Context, arg chat1.Download
 	if err != nil {
 		return res, err
 	}
+	if arg.MessageID == 0 {
+		return res, errors.New("invalid message ID provided")
```

Comment:
won't these just cause blackbars? why add them

Reply:
better error, I'm not sure what else to do here. I fixed the main cause on the JS I think.

Reply:
Seems reasonable to sanity check arguments early